src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -144,12 +143,12 @@ internal class TokenAcquisition : ITokenAcquisitionInternal
 
             try
             {
-                _application = GetOrBuildConfidentialClientApplication();
+                var application = BuildConfidentialClientApplication();
```

Comment:
@DavidParks8 The main issue @jmprieur  and I see here is that we could have 100s of tokenAcquisitions in the controller, and we won't want to recreate the cca object each time. Is there a reason why you removed `_application` and create it each time?

Reply:
Locking has been added back for lazy loading.